Userland/Libraries/LibWasm/Types.h

```diff
@@ -258,6 +258,11 @@ class Limits {
 
     auto min() const { return m_min; }
     auto& max() const { return m_max; }
+    bool matches(Limits other) const
```

Comment:
:thinking: is this wording from the spec?
I'd rather call this "is more limited than" or something in that vein

Reply:
Ah good point. How does is_subset_of work?